src/pages/sunggooKang/Login/LoginKang.js

```diff
@@ -1,8 +1,118 @@
-import React, { Component } from 'react';
+import React from 'react';
+import './LoginKang.scss';
+// import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+// import { restElement } from '@babel/types';
+
+class LoginKang extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      isBtnOn: false,
+      replies: [],
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    // const { id, pw } = this.state;
+
+    if (name === 'id') {
+      this.setState({
+        id: value,
+      });
+    } else {
+      this.setState({
+        pw: value,
+      });
```

Comment:
```suggestion
    this.setState({ [name]: value });
```
객체 계산된 속성명 사용하면 어떨까요?